ui/perfherder/compare/CompareSelectorView.jsx

```diff
@@ -130,7 +130,7 @@ export default class CompareSelectorView extends React.Component {
                 selectedRepo={originalProject}
                 title="Base"
                 checkbox
-                text="By default, Perfherder will compare against performance data gathered over the last 2 days from when new revision was pushed"
+                text="Tick off and you can choose a specific revision as a base. Otherwise, Perfherder will compare against performance data gathered over the last 2 days from when new revision was pushed"
```

Comment:
This is a good change.  Though "Tick off" may not be clear to all users.  In English, this can also mean "to make someone angry."  I think the wording might be better as:

  Uncheck this to choose a specific revision as a base.  Otherwise...

Would you mind updating it?

Reply:
Yeah, thank you! I will change it!